### DISCUSSION TITLE
Analytic mode for large set prior knowledge

### DISCUSSION THREAD

**Discussion by hbrehler** [asreview/asreview#1318] [Q&A] [answered]
Hi community!
For my review we first reviewed 1300 articles the old way, giving us quite a set of prior knowledge to feed ASReview (13.000 articles in our set in total). Does this influence the analytic settings/mode we have to choose  (e.g. set certainty-based to mixed), or are the standard settings still the best way to go?
We are performing a scoping review and our analysis will be qualitative primarily.

Love to hear from your
Heleen

**Comment by Rensvandeschoot** [comment, marked as answer]
You might be interested in reading more about the [Mega-Meta study](https://asreview.nl/project/the-megameta-project/)  in which we used labels from the first round of screening to train a neural network in the second round. This has shown to be a good strategy as shown in the [study](https://asreview.nl/project/switch-models/) by @JTeijema et al.

**Reply by hbrehler** [reply]
Thank you Rens, that is really helpful!
